docs/source/jit_language_reference_v2.rst

```diff
@@ -0,0 +1,321 @@
+.. contents::
+    :local:
+    :depth: 2
+
+
+.. testsetup::
+
+    # These are hidden from the docs, but these are necessary for `doctest`
+    # since the `inspect` module doesn't play nicely with the execution
+    # environment for `doctest`
+    import torch
+
+    original_script = torch.jit.script
+    def script_wrapper(obj, *args, **kwargs):
+        obj.__module__ = 'FakeMod'
+        return original_script(obj, *args, **kwargs)
+
+    torch.jit.script = script_wrapper
+
+    original_trace = torch.jit.trace
+    def trace_wrapper(obj, *args, **kwargs):
+        obj.__module__ = 'FakeMod'
+        return original_trace(obj, *args, **kwargs)
+
+    torch.jit.trace = trace_wrapper
+
+.. _language-reference:
```

Comment:
This is causing sphinx to choke with 
```
source/jit_language_reference_v2.rst:30: WARNING: duplicate label language-reference, other instance in /var/lib/jenkins/workspace/docs/source/jit_language_reference.rst
```

One of them should be renamed
@brianjo